src/Controls/samples/Controls.Sample/Pages/PlatformSpecifics/Windows/WindowsWebViewPage.xaml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
+
+namespace Maui.Controls.Sample.Pages
+{
+    public partial class WindowsWebViewPage : ContentPage
+    {
+        public WindowsWebViewPage()
+        {
+            InitializeComponent();
+
+            _webView.Source = new HtmlWebViewSource
+            {
+                Html = @"<html><body><button onclick=""window.alert('Hello World from JavaScript');"">Click Me</button></body></html>"
```

Comment:
this doesn't appear to work for me - I'm not seeing any alert pop up!

Reply:
This appears to be because webview2 has different JS apis to show popups. It used to be some JS `window.external.notify` and then subscribing to the `WebView.ScriptNotify` event, but now it is `chrome.window.sendMessage` and then listen on the `WebView2.CoreWebView2.WebMessageReceived` event.